benchmarks/dynamo/common.py

```diff
@@ -1521,6 +1530,10 @@ def run_one_model(
                 f"{stats['graph_breaks']} graph breaks ({stats['unique_graph_breaks']} unique)"
             )
 
+        if self.args.profile:
+            # TODO(voz): DO NOT LAND WITHOUT - Allow callers to define a file for this to write to
+            # Allow callers to define sort key
+            print(profiler.key_averages().table(sort_by="cpu_time_total"))
```

Comment:
@ezyang please bikeshed / opine on how you want this API to look, since this was PRd to help you investigate, you are patient 0 ;)

Reply:
In the end-to-end setup, I want these to show up in the generated performance CSVs, so that I can download them and look at them from the main CI runs.

Reply:
Hmm, that's kinda noisy for the profile CSVs! Do you want me to pull out the dynamo guards profile into the CSV... or all of it?

Reply:
If the metric is noisy, then IDK how helpful it is going to be (besides getting an order of magnitude estimate?) But if guards are really an overhead problem then it should take long enough to run them to cut through the noise, I'm guessing?